rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1600,20 +1600,20 @@ function createTable(gridId,data,columns, customParams, contextPath, refresh, st
     //add the two input fields for add, and a submit
     var pickStartInput = $('<input type="text">');
     pickStartInput.datetimepicker({dateFormat:'yy-mm-dd', timeFormat: 'HH:mm:ss', timeInput: true});
-    pickStartInput.addClass("pickStartInput");
-    $("#"+gridId+"_wrapper .dataTables_pickstart").append(pickStartInput);
+    pickStartInput.addClass("pickStartInput form-control input-xs");
+    $("#filterLogsStart").append(pickStartInput);
 
     var pickEndInput = $('<input type="text">');
     pickEndInput.datetimepicker({dateFormat:'yy-mm-dd', timeFormat: 'HH:mm:ss', timeInput: true});
-    pickEndInput.addClass("pickEndInput");
-    $("#"+gridId+"_wrapper .dataTables_pickend").append(pickEndInput);
+    pickEndInput.addClass("pickEndInput form-control input-xs");
+    $("#filterLogsEnd").append(pickEndInput);
 
-    var pickDatesButton = $("<button>Get Events Between Dates</button>");
+    var pickDatesButton = $("<button>Get Events</button>");
```

Comment:
Maybe we could have the button directly in html and only add the click function to "#filterLogsButton"

What do you think @RaphaelGauthier ?